x-pack/plugins/maps/public/classes/layers/solution_layers/security/security_layer_wizard.tsx

```diff
@@ -10,13 +10,30 @@ import { LAYER_WIZARD_CATEGORY } from '../../../../../common/constants';
 import { LayerWizard, RenderWizardArguments } from '../../layer_wizard_registry';
 import { getSecurityIndexPatterns } from './security_index_pattern_utils';
 import { SecurityLayerTemplate } from './security_layer_template';
+import { getUiSettings } from '../../../../kibana_services';
+
+function getIpId() {
+  const uiSettings = getUiSettings();
+  try {
+    const securityIndexPatternTitles = uiSettings.get('securitySolution:defaultIndex');
+    return securityIndexPatternTitles.join(',');
+  } catch (e) {
+    return '';
+  }
+}
 
 export const SecurityLayerWizardConfig: LayerWizard = {
   categories: [LAYER_WIZARD_CATEGORY.ELASTICSEARCH, LAYER_WIZARD_CATEGORY.SOLUTIONS],
-  checkVisibility: async () => {
+  getIsDisabled: async () => {
     const indexPatterns = await getSecurityIndexPatterns();
-    return indexPatterns.length > 0;
+    return indexPatterns.length === 0;
   },
+  disabledReason: i18n.translate('xpack.maps.security.disabledDesc', {
+    defaultMessage: 'There is no default security index {index}.',
```

Comment:
This disabled message is not clear enough to notify users what the problem is and how to resolve it. Maybe it should be something more like "Could not find security index pattern {index}.", Then add a sentence about how to get started with security.

Maybe @gchaps can help?

Reply:
You can add text that suggests users go to the Security Overview (which includes a link to the Security getting started guide.). Is the list of index patterns necessary?  My suggestion:

> Cannot find security index pattern.  To get started with Security, go to Security > Overview.

@miukimiu Is there a way to present this text outside of a tooltip, so we can link directly to the Security > Overview page?

Reply:
@gchaps a pattern that comes to my mind is the one we're using for the visualize modal. The card is disabled, but by clicking the badge, we can redirect users to a different page. 

<img width="860" alt="Screenshot 2021-01-13 at 17 48 07" src="https://user-images.githubusercontent.com/2750668/104489532-a3ce6f00-55c7-11eb-944b-f493f5992abb.png">

The badge could say something like **Disabled**. On hover the badge, the tooltip could say: "Cannot find security index pattern. Get started with Security".

What do you think @gchaps, @nreese, and @thomasneirynck? 

But then would we follow the same pattern for the other disabled cards?

Reply:
I am not sure if a user would expect/know that the badge is a clickable element.

Also, I think the badges emphasize the disabled cards, which should not be the case. If anything, the disabled cards should be deemphasized.

Honestly, if a user gets to this page and Security or Observability are not set up, I doubt that user would have the gumption/permissions to set up Security and Observability so a simple tooltip may be good enough. In the enterprise/corporate market, the user using the map is not going to be the same user setting up data flows and indexes.